src/api/cubrid_log.c

```diff
@@ -336,15 +337,20 @@ cubrid_log_connect_server_internal (char *host, int port, char *dbname)
   int err_code;
 
   char trace_errbuf[1024];
+  std::string msg;
 
   g_conn_entry = css_make_conn (INVALID_SOCKET);
   if (g_conn_entry == NULL)
     {
       CUBRID_LOG_ERROR_HANDLING (CUBRID_LOG_FAILED_CONNECT, trace_errbuf);
     }
 
+  //client should connect to Transaction server
+  msg = ((char) SERVER_TYPE_TRANSACTION) + '0';
+  msg.append (dbname, std::strlen (dbname) + 1);
   if (css_common_connect
-      (g_conn_entry, &rid, host, DATA_REQUEST, dbname, strlen (dbname) + 1, port, g_connection_timeout, true) == NULL)
+      (g_conn_entry, &rid, host, DATA_REQUEST, msg.c_str (), msg.length () + 1, port, g_connection_timeout,
```

Comment:
`strlen + 1` characters are already appended in msg. I don't think the extra character is needed here too.
```suggestion
      (g_conn_entry, &rid, host, DATA_REQUEST, msg.c_str (), msg.length (), port, g_connection_timeout,
```